src/content/self-hosted/administration/configuration.mdx

```diff
@@ -1120,6 +1120,9 @@ Enables [resource quotas](https://kubernetes.io/docs/concepts/policy/resource-qu
   - `maxReplicationControllers`: Maximum number of replication controllers per namespace.
   - `maxSecrets`: Maximum number of secrets per namespace.
   - `maxServices`: Maximum number of services per namespace.
+  - `maxIngresses`: Maximum number of ingresses per namespace.
```

Comment:
I think `maxIngresses` is already defined in line 1131 under `maxVolumeSnaphosts`